File: app/src/main/java/com/evaluation/model/search/SearchList.java
package com.evaluation.model.search;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class SearchList {

    @SerializedName("results")
    private List<SearchResult> searchResultList;

    public SearchList() {
    }

    public List<SearchResult> getSearchList() {
        return searchResultList;
    }
}

File: app/src/main/java/com/evaluation/model/search/SearchList.kt
package com.evaluation.model.search

import com.google.gson.annotations.SerializedName

/**
 * @author Vladyslav Havrylenko
 * @since 10.03.2020
 */
data class SearchList(
    @SerializedName("results")
    val searchResultList: List<SearchResult>
)